scripts/plot_dashboard.py

```
@@ -223,19 +223,27 @@ def plot_magnitude_phase(filt, sample_rate: float, dtypes: list[str] | None,
 
 def plot_pole_zero(filt):
     poles = np.asarray(filt.poles())
+    zeros = np.asarray(filt.zeros())
     theta = np.linspace(0.0, 2 * np.pi, 256)
     fig, ax = plt.subplots(figsize=(6, 6))
     ax.plot(np.cos(theta), np.sin(theta), color="0.6", linewidth=0.8)
     ax.axhline(0.0, color="0.85", linewidth=0.5)
     ax.axvline(0.0, color="0.85", linewidth=0.5)
+    # Conventional markers: × for poles, ○ for zeros. All-pole families
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)